samples/client/petstore-security-test/typescript-fetch/api.ts

```diff
@@ -115,11 +115,26 @@ export const FakeApiFp = {
  */
 export class FakeApi extends BaseAPI {
     /** 
-     * To test code injection *_/ &#39; \&quot; &#x3D;end \\r\\n \\n \\r
-     * @param test code inject * &#39; &quot; &#x3D;end rn n r To test code injection *_/ &#39; \&quot; &#x3D;end \\r\\n \\n \\r
+     * To test code injection *_/ &#39; \&quot; &#x3D;end -- \\r\\n \\n \\r
+     * @param test code inject * &#39; &quot; &#x3D;end  rn n r To test code injection *_/ &#39; \&quot; &#x3D;end -- \\r\\n \\n \\r
      */
-    testCodeInjectEndRnNR(params: {  test code inject * &#39; &quot; &#x3D;end rn n r?: string; }) {
-        return FakeApiFp.testCodeInjectEndRnNR(params)(this.fetch, this.basePath);
+    testCodeInjectEndRnNR(params: {  test code inject * &#39; &quot; &#x3D;end  rn n r?: string; }, options?: any) {
```

Comment:
This line confuse me. Can Typescript compile this? The added options part seems ok but does somebody know what is expected here as params?